### PR TITLE
feat: FS-2381

### PR DESCRIPTION
**Pull request by ckornell**
- Add Progress Bar Base Component
- Setup vNext for collaboration

### CODE REVIEW

File: packages/fscomponents/src/components/serializable/v2/SerializableProgressBar.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React, { FC, memo } from 'react';
```
Comment (ckornell) [inline packages/fscomponents/src/components/serializable/v2/SerializableProgressBar.tsx:1]: PR pertinent change

File: packages/fscomponents/src/components/index.ts [hunk 2]
```diff
@@ -78,3 +78,4 @@ export * from './LinkCard';
 
 export * as SerializableV0 from './serializable/v0';
 export * as SerializableV1 from './serializable/v1';
+export * as SerializableV2 from './serializable/v2';
```
Comment (ckornell) [inline packages/fscomponents/src/components/index.ts:81]: PR pertinent change

File: packages/fscomponents/src/components/index.ts [hunk 3]
```diff
@@ -78,3 +78,4 @@ export * from './LinkCard';
 
 export * as SerializableV0 from './serializable/v0';
 export * as SerializableV1 from './serializable/v1';
+export * as SerializableV2 from './serializable/vNext';
```
Comment (wSedlacek) [inline packages/fscomponents/src/components/index.ts:81]:
```suggestion
export * as SerializableVNext from './serializable/vNext';
```

**Review by wSedlacek — APPROVED** [review]
LGTM

**Review by bweissbart — APPROVED** [review]

**Pull request merged**